docs/basics/acct-team-mgmt/concurrency/managing-concurrency.md

```diff
@@ -57,3 +63,29 @@ curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
 --request GET 'https://api.us-west-1.saucelabs.com/rest/v1.2/users/<username>/concurrency' \
 --header 'Content-Type: application/json' \ | json_pp
 ```
+
+### Exceeding Virtual Cloud Concurrency
+
+The Virtual Cloud is used to run business critical pipelines across large enterprise organizations. This means there will be times when it is difficult to predict exactly how much concurrency you will need at all times. For these situations we allow your organization to exceed it's concurrency limit.
+
+The following table describes the maximum amount you are allowed to exceed your concurrency limit.
+
+| Subscription Amount | % You Can Exceed | Example | 
+|---------------------|-----------------|---------|
+|1-100| 200%| 50 -> 150|
+|101-500|100%|200 -> 400|
+|501-2000|50%|1000 -> 1500|
+|2001+|10%|2500 -> 2750|
+
+Once your organization attempts to run a test past the maximum amount you can exceed your tests will receive a [You've Exceeding Your Concurrency Limit Error](/dev/error-messages/#youve-exceeded-your-sauce-labs-concurrency-limit)
+
+## Real Device Cloud Concurrency
+
+If your subscription includes access to our Public Real Device Cloud, then your subscription quantity equals the amount of concurrent tests that can be run on Sauce Labs real devices.
```

Comment:
@yuyi-sl could we add a cross link here for private devices? "if you need to manage your private device assignments, go to this link." 
Some folks might stumble open this page first before that